### PR TITLE
Upgrade to .NET 7, enable WASM optimizations

### PR DESCRIPTION
**Pull request by Arlodotexe**
Companion PR to https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/117, which should be closed before merging this.

As noted:
> Any codebase consuming the tooling repo will need to be adjusted to account for the new nullable annotations and analyzers added between netstandard2.0 and net7.0.

This PR pulls in the .NET 7 upgrade and fixes the new warnings and errors that appeared in our codebase as a result.

### CODE REVIEW

**Comment by Arlodotexe** [comment]
Local packages were built for this branch tested to work on these platforms:
- [x] Uwp
- [x] WinAppSdk
- [x] WebAssembly
- [x] Linux
- [x] WPF
- [ ] MacOS
- [ ] iOS
- [ ] Android <details><summary>Runtime error, see details.</summary>

  image
  
            > System.InvalidOperationException: 'Failed to load ToolkitPackageUwpUnoTest.MainPage: Java.Lang.ClassNotFoundException: crc643edf5f81ce7abab9.Segmented
            > ---> Java.Lang.ClassNotFoundException: Didn't find class "crc643edf5f81ce7abab9.Segmented" on path: DexPathList[[zip file "/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk"],nativeLibraryDirectories=[/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/lib/x86_64, /data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
            > --- End of managed Java.Lang.ClassNotFoundException stack trace ---
            > java.lang.ClassNotFoundException: Didn't find class "crc643edf5f81ce7abab9.Segmented" on path: DexPathList[[zip file "/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk"],nativeLibraryDirectories=[/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/lib/x86_64, /data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
            > 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
            > 	at java.lang.Class.classForName(Native Method)
            > 	at java.lang.Class.forName(Class.java:454)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.n_onActivityStarted(Native Method)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.onActivityStarted(NativeApplication_ActivityCallbacks.java:99)
            > 	at android.app.Application.dispatchActivityStarted(Application.java:401)
            > 	at android.app.Activity.dispatchActivityStarted(Activity.java:1409)
            > 	at android.app.Activity.onStart(Activity.java:1948)
            > 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:496)
            > 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:246)
            > 	at crc6467d88aaa7548788d.BaseActivity.n_onStart(Native Method)
            > 	at crc6467d88aaa7548788d.BaseActivity.onStart(BaseActivity.java:611)
            > 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1543)
            > 	at android.app.Activity.performStart(Activity.java:8367)
            > 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3669)
            > 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:224)
            > 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:204)
            > 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
            > 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
            > 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2306)
            > 	at android.os.Handler.dispatchMessage(Handler.java:106)
            > 	at android.os.Looper.loopOnce(Looper.java:201)
            > 	at android.os.Looper.loop(Looper.java:288)
            > 	at android.app.ActivityThread.main(ActivityThread.java:7918)
            > 	at java.lang.reflect.Method.invoke(Native Method)
            > 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
            > 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
            > --- End of managed Java.Lang.ClassNotFoundException stack trace ---
            > java.lang.ClassNotFoundException: Didn't find class "crc643edf5f81ce7abab9.Segmented" on path: DexPathList[[zip file "/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk"],nativeLibraryDirectories=[/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/lib/x86_64, /data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
            > 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
            > 	at java.lang.Class.classForName(Native Method)
            > 	at java.lang.Class.forName(Class.java:454)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.n_onActivityStarted(Native Method)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.onActivityStarted(NativeApplication_ActivityCallbacks.java:99)
            > 	at android.app.Application.dispatchActivityStarted(Application.java:401)
            > 	at android.app.Activity.dispatchActivityStarted(Activity.java:1409)
            > 	at android.app.Activity.onStart(Activity.java:1948)
            > 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:496)
            > 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:246)
            > 	at crc6467d88aaa7548788d.BaseActivity.n_onStart(Native Method)
            > 	at crc6467d88aaa7548788d.BaseActivity.onStart(BaseActivity.java:611)
            > 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1543)
            > 	at android.app.Activity.performStart(Activity.java:8367)
            > 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3669)
            > 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:224)
            > 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:204)
            > 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
            > 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
            > 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2306)
            > 	at android.os.Handler.dispatchMessage(Handler.java:106)
            > 	at android.os.Looper.loopOnce(Looper.java:201)
            > 	at android.os.Looper.loop(Looper.java:288)
            > 	at android.app.ActivityThread.main(ActivityThread.java:7918)
            > 	at java.lang.reflect.Method.invoke(Native Method)
            > 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
            > 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
            > --- End of inner exception stack trace ---
            > at Java.Interop.JniEnvironment.StaticMethods.CallStaticObjectMethod(JniObjectReference type, JniMethodInfo method, JniArgumentValue* args) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/obj/Release/net7.0/JniEnvironment.g.cs:line 12890
            > at Android.Runtime.JNIEnv.FindClass(String classname) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNIEnv.cs:line 577
            > at Android.Runtime.JNIEnv.FindClass(Type type) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNIEnv.cs:line 532
            > at Uno.UI.NativeInstanceHelper.CreateTypeReference(Type type) in C:\a\1\s\src\Uno.UI\Extensions\NativeViewInstanceHelper.Android.cs:line 121
            > at Uno.UI.NativeInstanceHelper.CreateNativeInstance(Type type, Object target, Context context, Action`2 setInstance) in C:\a\1\s\src\Uno.UI\Extensions\NativeViewInstanceHelper.Android.cs:line 97
            > at Uno.UI.Controls.BindableView..ctor(Context context) in C:\a\1\s\src\Uno.UI\Controls\BindableView.Android.cs:line 48
            > at Windows.UI.Xaml.UIElement..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\UIElement.Android.cs:line 84
            > at Windows.UI.Xaml.FrameworkElement..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\FrameworkElement.Android.cs:line 29
            > at Windows.UI.Xaml.Controls.Control..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Control\Control.Android.cs:line 9
            > at Windows.UI.Xaml.Controls.ItemsControl..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ItemsControl\ItemsControl.cs:line 83
            > at Windows.UI.Xaml.Controls.Primitives.Selector..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Primitives\Selector.cs:line 54
            > at Windows.UI.Xaml.Controls.ListViewBase..ctor() in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\ListViewBase\ListViewBase.cs:line 55
            > at CommunityToolkit.WinUI.Controls.Segmented..ctor() in /_/components/Segmented/src/Segmented/Segmented.cs:line 14
            > at ToolkitPackageUwpUnoTest.MainPage.InitializeComponent_C485E415() in D:\source\uwp\app_\ToolkitPackageUwpUnoTest\ToolkitPackageUwpUnoTest.Mobile\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MainPage_ba262bb41580a9a89dedc5da47a1ff49.cs:line 72
            > at ToolkitPackageUwpUnoTest.MainPage.InitializeComponent() in D:\source\uwp\app_\ToolkitPackageUwpUnoTest\ToolkitPackageUwpUnoTest.Mobile\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MainPage_ba262bb41580a9a89dedc5da47a1ff49.cs:line 58
            > at ToolkitPackageUwpUnoTest.MainPage..ctor() in D:\source\uwp\app_\ToolkitPackageUwpUnoTest\ToolkitPackageUwpUnoTest.Shared\MainPage.xaml.cs:line 27
            > at ToolkitPackageUwpUnoTest.Mobile.MetadataBuilder_757.CreateInstance() in D:\source\uwp\app_\ToolkitPackageUwpUnoTest\ToolkitPackageUwpUnoTest.Mobile\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.BindableTypeProviders.BindableTypeProvidersSourceGenerator\BindableMetadata.g.cs:line 30743
            > at Windows.UI.Xaml.Controls.Frame.CreatePageInstance(Type sourcePageType) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Frame\Frame.cs:line 454
            > at Windows.UI.Xaml.PagePool.DequeuePage(Type pageType) in C:\a\1\s\src\Uno.UI\UI\Xaml\PagePool.cs:line 69
            > at Windows.UI.Xaml.Controls.Frame.CreatePageInstanceCached(Type sourcePageType) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Frame\Frame.cs:line 444
            > at Windows.UI.Xaml.Controls.Frame.EnsurePageInitialized(PageStackEntry entry) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Frame\Frame.cs:line 435
            > at Windows.UI.Xaml.Controls.Frame.InnerNavigate(PageStackEntry entry, NavigationMode mode) in C:\a\1\s\src\Uno.UI\UI\Xaml\Controls\Frame\Frame.cs:line 283
            > --- End of managed Java.Lang.ClassNotFoundException stack trace ---
            > java.lang.ClassNotFoundException: crc643edf5f81ce7abab9.Segmented
            > 	at java.lang.Class.classForName(Native Method)
            > 	at java.lang.Class.forName(Class.java:454)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.n_onActivityStarted(Native Method)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.onActivityStarted(NativeApplication_ActivityCallbacks.java:99)
            > 	at android.app.Application.dispatchActivityStarted(Application.java:401)
            > 	at android.app.Activity.dispatchActivityStarted(Activity.java:1409)
            > 	at android.app.Activity.onStart(Activity.java:1948)
            > 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:496)
            > 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:246)
            > 	at crc6467d88aaa7548788d.BaseActivity.n_onStart(Native Method)
            > 	at crc6467d88aaa7548788d.BaseActivity.onStart(BaseActivity.java:611)
            > 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1543)
            > 	at android.app.Activity.performStart(Activity.java:8367)
            > 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3669)
            > 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:224)
            > 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:204)
            > 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
            > 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
            > 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2306)
            > 	at android.os.Handler.dispatchMessage(Handler.java:106)
            > 	at android.os.Looper.loopOnce(Looper.java:201)
            > 	at android.os.Looper.loop(Looper.java:288)
            > 	at android.app.ActivityThread.main(ActivityThread.java:7918)
            > 	at java.lang.reflect.Method.invoke(Native Method)
            > 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
            > 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
            > Caused by: java.lang.ClassNotFoundException: Didn't find class "crc643edf5f81ce7abab9.Segmented" on path: DexPathList[[zip file "/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk"],nativeLibraryDirectories=[/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/lib/x86_64, /data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
            > 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
            > 	... 26 more
            > --- End of managed Java.Lang.ClassNotFoundException stack trace ---
            > java.lang.ClassNotFoundException: crc643edf5f81ce7abab9.Segmented
            > 	at java.lang.Class.classForName(Native Method)
            > 	at java.lang.Class.forName(Class.java:454)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.n_onActivityStarted(Native Method)
            > 	at crc64122dcf5ad656876d.NativeApplication_ActivityCallbacks.onActivityStarted(NativeApplication_ActivityCallbacks.java:99)
            > 	at android.app.Application.dispatchActivityStarted(Application.java:401)
            > 	at android.app.Activity.dispatchActivityStarted(Activity.java:1409)
            > 	at android.app.Activity.onStart(Activity.java:1948)
            > 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:496)
            > 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:246)
            > 	at crc6467d88aaa7548788d.BaseActivity.n_onStart(Native Method)
            > 	at crc6467d88aaa7548788d.BaseActivity.onStart(BaseActivity.java:611)
            > 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1543)
            > 	at android.app.Activity.performStart(Activity.java:8367)
            > 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3669)
            > 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:224)
            > 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:204)
            > 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
            > 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
            > 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2306)
            > 	at android.os.Handler.dispatchMessage(Handler.java:106)
            > 	at android.os.Looper.loopOnce(Looper.java:201)
            > 	at android.os.Looper.loop(Looper.java:288)
            > 	at android.app.ActivityThread.main(ActivityThread.java:7918)
            > 	at java.lang.reflect.Method.invoke(Native Method)
            > 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
            > 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
            > Caused by: java.lang.ClassNotFoundException: Didn't find class "crc643edf5f81ce7abab9.Segmented" on path: DexPathList[[zip file "/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk"],nativeLibraryDirectories=[/data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/lib/x86_64, /data/app/~~6pQFnZX-1E4rY07w1f_phw==/ToolkitPackageUwpUnoTest.ToolkitPackageUwpUnoTest-gZ5Gb9jmLleNqZsxdFVgkg==/base.apk!/lib/x86_64, /system/lib64, /system_ext/lib64]]
            > 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
            > 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
            > 	... 26 more
  
         </details>

File: .editorconfig
Comment (michael-hawker) [inline .editorconfig:440]: We'll want to resolve these before merging.

Comment (Arlodotexe) [reply .editorconfig]: Oddly enough, they only show on net7-android, and so far I've been unable to repro locally.

The pull request packages will be available when the CI passes, I agree let's resolve this before merging.

**Comment by michael-hawker** [comment]
Kicked off a new build, hopefully the package versioning stuff is all resolved, so we'll have a clean set of new packages to try soon! 🤞

**Comment by michael-hawker** [comment]
Build good, but have been having trouble with the uno template from VS on my machine, trying in VS Code now. Got this message from NuGet about the android package:

> C:\temp\UnoTestApp\UnoTestApp\UnoTestApp.csproj : error NU1202: Package CommunityToolkit.WinUI.Controls.Primitives 8.0.230815-pull-173.526 is not compatible with ne
t7.0-android33.0 (.NETCoreApp,Version=v7.0). Package CommunityToolkit.WinUI.Controls.Primitives 8.0.230815-pull-173.526 supports: [C:\temp\UnoTestApp\UnoTestApp.Win
dows\UnoTestApp.Windows.csproj]

Do we have to be more specific than `net7.0-android`?

**Comment by michael-hawker** [comment]
Was able to test in VS Code from the command line with `dotnet run` that the WASM side of the WinUI Primitives package works fine! 🎉🎉🎉 Will see if I can get anything else working, maybe with XAML too.

**Comment by michael-hawker** [comment]
Tested the `Uwp` packages for both WinUI 2 and 3 for the Segmented control package in release mode, worked fine.

We can probably remove the `all-uwp` restriction from the WASDK build. Seems like things were fixed in 1.3. Means the only use of the packages is really between Uno.UI vs. Uno.WinUI for Uno Platform. Would make migrating easier for folks potentially as they can transform their UWP to WASDK without having to change the Toolkit packages just yet. We can just preface and say that things may change in the future type thing, as required and that eventually once Uno Platform stops support Uno.UI we'd only have the one package (unless we end up having to separate them again due to changes in WASDK/WinUI 3).

**Comment by Arlodotexe** [comment]
> Do we have to be more specific than net7.0-android?

Found some useful information about how Uno handles net6 and net7 targets on Android: https://github.com/unoplatform/uno/discussions/11423
I've also reached out to Uno via Discord: https://discord.com/channels/372137812037730304/1139681165142396948/1141422502304821258

I'll switch us over to android33.0 for now and trying that while waiting for a reply.

**Comment by michael-hawker** [comment]
Looks like these are the last warnings hopefully:

```
D:\a\Windows\Windows\components\Animations\src\Enums\FrameworkLayer.cs(16,5): error CS1587: XML comment is not placed on a valid language element [D:\a\Windows\Windows\components\Animations\src\CommunityToolkit.WinUI.Animations.csproj::TargetFramework=net7.0-android33.0]
         D:\a\Windows\Windows\components\Animations\src\Enums\FrameworkLayer.cs(12,5): error CS1587: XML comment is not placed on a valid language element [D:\a\Windows\Windows\components\Animations\src\CommunityToolkit.WinUI.Animations.csproj::TargetFramework=net7.0-android33.0]
         D:\a\Windows\Windows\components\Animations\src\Enums\FrameworkLayer.cs(25,5): error CS1587: XML comment is not placed on a valid language element [D:\a\Windows\Windows\components\Animations\src\CommunityToolkit.WinUI.Animations.csproj::TargetFramework=net7.0-android33.0]
         D:\a\Windows\Windows\components\Animations\src\Enums\FrameworkLayer.cs(21,5): error CS1587: XML comment is not placed on a valid language element [D:\a\Windows\Windows\components\Animations\src\CommunityToolkit.WinUI.Animations.csproj::TargetFramework=net7.0-android33.0]
         D:\a\Windows\Windows\components\Animations\src\Xaml\Abstract\CustomAnimation{TValue,TKeyFrame}.cs(22,5): error CS1587: XML comment is not placed on a valid language element [D:\a\Windows\Windows\components\Animations\src\CommunityToolkit.WinUI.Animations.csproj::TargetFramework=net7.0-android33.0]
```

File: .github/workflows/build.yml
Comment (michael-hawker) [inline .github/workflows/build.yml:20]: Ah, mentioned this on the tooling PR, we should open an issue over there to look at this later.

File: components/Animations/src/Enums/FrameworkLayer.cs
Comment (michael-hawker) [inline components/Animations/src/Enums/FrameworkLayer.cs:17]: We could put this at the top with the using:
```csharp
#if WINUI2
using Composition = Windows.UI.Composition;
#elif WINUI3
using Composition = Microsoft.UI.Composition;
#endif
```
And then just be able to do `Composition` in the comment vs. duplicating all the text, same with below and anywhere else. Makes it easier to maintain the comment in the future.

**Review by michael-hawker — APPROVED** [review]
Small nit, and something to follow-up on later, but otherwise we should be good. Yay!

Did these not need to be addressed too though? https://github.com/CommunityToolkit/Windows/pull/173#issuecomment-1682569345

**Pull request merged**
